### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by catap** [uTox/uTox#1566]
Hey,

Does this project alive?

**Comment by Neustradamus** [comment]
@catap: Good question!

**Comment by catap** [comment]
@Neustradamus what I see:
1. The last meaningful commit to the `develop' branch was Jan 28, 2021 (86bdb6f85d663f0ff0c6b65ac150bd57ceeaf9b1).
2. The last meaningful commit to the repository was into `new_groupchats` brnach on Mar 14, 2022 (fa88927682f595ec3a3bd9711465b956de0bc7da) as https://github.com/uTox/uTox/pull/1271.
3. It contains many open issues like "crash on ..." and "segfault in ...".
4. My attemp to use it linked against toxcore to 0.2.19 leads to corruption of `~/.config/tox` by creating some file with wired names and brokes profile itself.
5. Probably soon https://github.com/TokTok/c-toxcore/pull/2450 will be finished, that makes it quite impossible to use clients with old version of toxcore.

Everything seems to lead to a conclusion that this project is dead and dangerous to use.

**Comment by catap** [comment]
@barracuda156 X11, but it doesn't change anything.

**Comment by barracuda156** [comment]
@catap BTW, do you know a macOS replacement for `ld -r -b binary`? I have kinda fixed the build for powerpc today, but need that thing for tray to work, otherwise it crashes :) 
It is used here: https://github.com/uTox/uTox/blob/ce4b44a10b5a9866f4be852b8dcbcb264ab6cd01/src/xlib/CMakeLists.txt#L16
Just trashing the chunk does not really work since the code in `tray.c` uses these symbols then.

**Comment by catap** [comment]
@barracuda156 I have no idea.

And the bad news that it may broke your tox profile.

**Comment by barracuda156** [comment]
> qtox is being revived by some of the core devs. https://github.com/TokTok/qTox

@lotsm-unix While that is certainly welcome, qTox requires Qt5, which is heavy and broken on some non-mainstream platforms.

**Comment by catap** [comment]
> qtox is being revived by some of the core devs. https://github.com/TokTok/qTox

WoW

let see how it goes in a year.

**Comment by raphael10-collab** [comment]
I agree with @barracuda156 . qTox relies on Qt5, meaning it is not platform-agnostic and independent. While uTox is, or better say, unfortunately !, was, independent from any platform

**Comment by barracuda156** [comment]
@lotsm-unix That does not make situation any worse as compared to Qt5, at least for me, and _in principle_ it may be even easier to fix Qt6 rather than Qt5 (at least due to a sane build system finally), but in any case, for now that is not a workable option.

**Comment by Neustradamus** [comment]
Dear @uTox team, @GrayHatter, @publicarray, @robinlinden, @jhert0,

Can you add new team members, it is important to have a continuity of the uTox project.

Thanks in advance.

**Comment by GrayHatter** [comment]
@Neustradamus I have no plans to add anyone as a repo maintainer, only because there's no one who's contributed enough to become trusted. I see @barracuda156 has a draft pr #1575 to fix osx, but I have no way to build or test OSX, so I can't review or merge it myself. 

Still happy to merge any changes that can be reviewed, tested, and approved by more than one person. If they're then able to continue to meaningfully contribute, I'd be happy to include anyone interested as a maintainer.

**Comment by barracuda156** [comment]
@GrayHatter To be clear, I was not asking to be a maintainer, I merely want to improve support for macOS and, hopefully, get it actually running on PowerPC.

**Comment by GrayHatter** [comment]
@barracuda156 yeah, I totally got that. It's impossible for me to express my actual appreciation for your PR, especially given I can't test it myself or merge it without someone else verifying the same. I was trying very hard not to volunteer *anyone* for the extra work. While also making it clear I would consider it :)
